Project/Code/enabled_gems.cmake

```diff
@@ -1,6 +1,5 @@
 
 set(ENABLED_GEMS
-    Buongiorno
     Atom
```

Comment:
The correct gem name is below, had to remove this unused one to prevent a warning when registering project